packages/model-viewer/src/three-components/gltf-instance/correlated-scene-graph.ts

```diff
@@ -240,49 +240,33 @@ export class CorrelatedSceneGraph {
     this[$threeObjectMap] = threeObjectMap;
   }
 
-  loadVariant(variantIndex: number, onUpdate: () => void = () => {}):
-      Set<number> {
+  async loadVariant(variantName: string, onUpdate: () => void = () => {}):
+      Promise<Set<number>> {
```

Comment:
Two changes
* Taking `variantName`, not `variantIndex`, because `variantName` needs to be passed to the plugin function
* Changed from sync to async method because `updateMaterials` (see below) are asynchronously updated

Reply:
Perfect, thanks! And this is not a public API change anyway because this is a private class (all instances hidden behind symbols), so no worries.